source/install/docker-upgrading.rst

```diff
@@ -1,146 +1,128 @@
 
-.. include:: ../.res/code/docker.stop.pmm-server.txt
+      The output is a JSON string similar to this one.
 
-The following command simply renames the current |opt.pmm-server| container to
-avoid name conflicts during the update procedure:
+      .. code-block:: text
 
-.. include:: ../.res/code/docker.rename.pmm-server.pmm-server-backup.txt
+         {"version":"2.9.0","server":{"version":"2.9.0","full_version":"2.9.0-37.2007140859.f31f32e.el7","timestamp":"2020-07-14T08:59:38Z"},"managed":{"version":"2.9.0","full_version":"4b39891989e04e1c33f8d78bb3fd674be6705bb7","timestamp":"2020-07-13T15:37:59Z"},"distribution_method":"DOCKER"}%
 
-.. _pmm.deploying.docker-image.pulling:
+2. Check if there is a newer version of PMM Server.
 
-`Pulling a new Docker Image <docker-upgrading.html#image-pulling>`_
---------------------------------------------------------------------------------
+   Visit `<https://hub.docker.com/r/percona/pmm-server/tags/>`_.
 
-|docker| images for all versions of |pmm| are available from
-`percona/pmm-server`_
-|docker| repository.
+3. Stop the container and create backups.
 
-When pulling a newer |docker| image, you may either use a specific version
-number or the ``2`` image which always matches the highest version
-number. 
+   Back-up the current container and its data so that
+   you can revert back to using them, and as a safeguard in case
+   the update procedure fails.
 
-This example shows how to pull a specific version:
+   .. code-block:: bash
 
-.. code-block:: bash
+      docker stop pmm-server
+      docker rename pmm-server pmm-server-backup
+      docker cp pmm-data pmm-data-backup
 
-   $ docker pull percona/pmm-server:2.8.0
+4. Pull the new PMM Server image.
 
-This example shows how to pull the latest PMM 2 version:
-   
-.. include:: ../.res/code/docker.pull.percona-pmm-server-latest.txt
-   
-.. _pmm.deploying.docker-container.creating:
+   You may specify an exact version number, or the latest.
 
-`Creating a new Docker container based on the new image <docker-upgrading.html#container-creating>`_
--------------------------------------------------------------------------------------------------------
+   To pull a specific version (2.9.0 in this example):
 
-After you have pulled a new version of |pmm| from the |docker| repository, you can
-use |docker.run| to create a |opt.pmm-server| container using the new image.
+   .. code-block:: bash
 
-.. include:: ../.res/code/docker.run.latest.txt
+      docker pull percona/pmm-server:2.9.0
 
-.. important::
+   To pull the latest version of PMM 2:
 
-   The |opt.pmm-server| container must be stopped before attempting |docker.run|.
+   .. code-block:: bash
 
-The |docker.run| command refers to the pulled image as the last parameter. If
-you used a specific version number when running |docker.pull| (see
-:ref:`pmm.server.docker-image.pulling`) replace ``2`` accordingly.
+      docker pull percona/pmm-server:2
```

Comment:
Or pmm-server:latest?

Reply:
it is not recommended by docker to use tag latest in your environment.